### PR TITLE
Added Avatars

### PR DESCRIPTION
**Pull request by SukanyaSen530**
-Added Avatars
   a. Image Avatars
   b. Text Avatars
   c.  Icon Avatars
   d. Different Sizes - sm (3rem), md (4rem), lg(5rem)
   e. Square Avatars

Avatars

### CODE REVIEW

File: docs.html [hunk 1]
```diff
@@ -270,26 +270,126 @@ <h6>H6-Heading</h6>
...
-      <p></p>
+      <p>Avatar are images which can be used to show user's profile image. Use base class <span class="code-highlight">
+          .avatar</span> to generate a avatar.</p>
+
+      <h5>Image Avatars</h5>
+      <p>Image avatars can be created with class <span class="code-highlight"> .avatar-img</span></p>
+
+      <div class="component-display">
+        <figure class="avatar avatar-md">
+          <img class="avatar-img" src="./assets/images/rengoku.jpg" alt="">
```
Comment (dev-enforced) [inline docs.html]: Add an alt attribute to the images used in documentation.In this case you can mention anime-character or Demo Image😅.

**Review by ArjunGTX — APPROVED** [review]
Looks good. Great that you used variables for spacing too.

File: ht.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+    <figure class="avatar avatar-letter avatar-sm warning ">
+        <span>RK</span>
+    </figure>
+
+    <!-- Icon Avatar -->
+
+    <!-- SVG Image  -->
+    <figure class="avatar avatar-icon  avatar-sm primary">
+        <img class="avatar-icon" src="./camera.svg" alt="">
```
Comment (tanishk15sharma) [inline ht.html]: give a short info in  `alt` everywhere

File: ht.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+
```
Comment (dev-enforced) [inline ht.html]: Had a small clarification: Is this html file just the compilation of all HTML code which you have written to describe the avatars?

**Review by tanishk15sharma — APPROVED** [review]

**Review by dev-enforced — APPROVED** [review]
Code looks really good to me!!Great work Sukanya on the avatars.Variables and naming conventions are really good.Some small changes mentioned below.

Do deploy this on Netlify or Vercel so that the reviewers can view the changes on code as well as the resulting output.

File: ht.html (hunk 2 above)
Comment (SukanyaSen530) [reply ht.html]: I forgot to remove that file, that is for testing. thanks

Comment (SukanyaSen530) [reply ht.html]: fixed - d024da009973ecafa0fe044faac36624c2778267

File: ht.html (hunk 3 above)
Comment (SukanyaSen530) [reply ht.html]: Yes, that was for testing, I removed it

fixed - d024da009973ecafa0fe044faac36624c2778267

File: docs.html (hunk 1 above)
Comment (SukanyaSen530) [reply docs.html]: fixed - d024da009973ecafa0fe044faac36624c2778267

**Review by VedanthB — APPROVED** [review]
looks good to go 🚀

File: ht.html (hunk 3 above)
Comment (dev-enforced) [reply ht.html]: Got it Sukanya!!You can merge it.

**Pull request merged**